tests/fXmlRpc/Parser/AbstractParserTest.php

```diff
@@ -697,4 +699,25 @@ public function testEmptyStructMember()
         $this->assertSame(array('FIRST' => ''), $this->parser->parse($string, $isFault));
         $this->assertFalse($isFault);
     }
+
+    public function testThrowExceptionWhenIsString()
+    {
+        $string = 'returned string';
+
+        $this->setExpectedException(
+            'fXmlRpc\Exception\ParserException',
+            'Invalid XML. Expected XML, string given: "returned string"'
+        );
+        $this->parser->parse($string, $isFault);
+    }
+
+    public function testNovalidateWhenResponseIsString()
+    {
+        $string = 'returned string';
+
+        $parser = $this->getParserWithoutValidation();
```

Comment:
The test case should have a "protected abstract function createParserWithoutValidation()"